### DISCUSSION TITLE
Problem with link on a table row

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6306] [Help] [answered]
Hello, 
I do not manage to have a link working if I put it on the table row.
The target page is `/questions/2-nouvelle-question/{{ tbl_liste_articles.base_article.Code article }}`
If I put it on a link element , it works.
If I put it on the table row, it sends me to another page of the app (new row page of another object...)
Is it a known bug ?

**Comment by melohagan** [comment, marked as answer]
Hi @NicolasFabre81 

This may help: https://docs.budibase.com/docs/url-parameters#link-navigation

**Reply by NicolasFabre81** [reply]
Already read this but... Sometimes we need two times ! Thanks

**Reply by melohagan** [reply]
No worries!
